rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -919,21 +919,22 @@ private void processFundsMigration(Transaction rskTx) throws IOException {
         List<UTXO> availableUTXOs = getRetiringFederationBtcUTXOs();
         Federation activeFederation = getActiveFederation();
 
-        logger.trace("[processFundsMigration] Going to process migration...");
-
-        if (federationIsInMigrationAge(activeFederation) && hasMinimumFundsToMigrate(retiringFederationWallet)) {
-            logger.info(
-                "[processFundsMigration] Active federation (age={}) is in migration age and retiring federation has funds to migrate: {}.",
-                rskExecutionBlock.getNumber() - activeFederation.getCreationBlockNumber(),
-                retiringFederationWallet.getBalance().toFriendlyString()
-            );
+        if (federationIsInMigrationAge(activeFederation)) {
+            long migrationAge = rskExecutionBlock.getNumber() - activeFederation.getCreationBlockNumber();
+            logger.trace("[processFundsMigration] Active federation (age={}) is in migration age.", migrationAge);
+            if (hasMinimumFundsToMigrate(retiringFederationWallet)){
```

Comment:
nit:
Shouldn't we log if when the condition is false? maybe saying
```
[processFundsMigration] Retiring federation doesn't have funds to migrate: {}.
```

Reply:
The thing is that it most likely will flood the logs with no useful information. Why? because the method `updateCollections` is called many times, which then calls `processFundsMigration` each time, and as the migration of funds is only done when the above conditions are met, which is something that only happens during a migration, it will be logging `Retiring federation doesn't have funds to migrate: {}.` almost all the time.

Reply:
Ok, sounds reasonable to me.

Thanks for explaning.